tests/rest/client/v2_alpha/test_relations.py

```diff
@@ -628,7 +629,7 @@ def _send_relation(
         relation_type,
         event_type,
         key=None,
-        content={},
+        content: Optional[dict] = None,
```

Comment:
In the docstring below, `content` is possibly `None`, however the default value was always `{}` instead of `None`, should something be changed about this (the type, the docstring, etc.), or is this fine just as it is annotated right now?

Reply:
Test failures show that this should be `= {}`, docstring below is just what you can put in, which makes sense, i guess.